sdk/storage/azure-storage-blob-changefeed/src/main/java/com/azure/storage/blob/changefeed/BlobChunkedDownloader.java

```diff
@@ -52,7 +52,8 @@ public Flux<ByteBuffer> download() {
             -> client.downloadWithResponse(range, null, conditions, false);
 
         /* We don't etag lock since the Changefeed can append to the blob while we are reading it. */
-        return ChunkedDownloadUtils.downloadFirstChunk(range, options, requestConditions, downloadFunc, false)
+        return ChunkedDownloadUtils.downloadFirstChunk(range, options, requestConditions, downloadFunc, false,
+            null)
```

Comment:
In the `ChunkedDownloadUtils` class, I'm wondering if it makes sense to introduce this as an overloaded method? Mainly this will:
1. Allow us from avoiding needing to update the call pattern everywhere the chunked downloader class is used but no context is need to be provided
2. Help avoid conflict mismatches in clients (e.g. the customer has an older version of the blob client that does not support this method signature)

I'm not up to speed on how this is typically handled between packages so would be happy to hear thoughts on this idea.